noir-projects/aztec-nr/aztec/src/context/avm_context.nr

```diff
@@ -123,7 +123,7 @@ impl PublicContextInterface for AvmContext {
         temporary_function_selector: FunctionSelector,
```

Comment:
I think this method mirrors the current method in `public_context`. Do you know what's the plan with the hardcoded values? Would the idea be that we'd change them to `context.XXXgasLeft()`?

Or should the `call` oracle just not have any gas arguments and internally (in TS) use the gas left?

Reply:
I'd expect it to work like in Ethereum-land: the EVM requires you to be explicit about how much you pass to the internal call, but Solidity defaults to passing all gasLeft(). I can look into this in a later PR.